cmd/network/verification.go

```diff
@@ -790,3 +801,60 @@ func printVersion() {
 	}
 	log.Println(fmt.Sprintf("Using osd-network-verifier version %v", version))
 }
+
+// Put Cluster to LS if it can't access pd (OR) DMS
+func putLimitedSupport(ClusterId string) error {
```

Comment:
**chore:** As mentioned above, I suggest re-using the existing LS post command to avoid code duplication.

Reply:
reused post's struct to set cluster into LS. Removed all re-used code